src/Lumi/Styles/Button.purs

```diff
@@ -177,7 +177,7 @@ button colo kind state size = case kind of
               , minWidth: int 70
               , padding: str "10px 20px"
               , fontSize: int 14
-              , lineHeight: int 20
+              , lineHeight: int 1
```

Comment:
Nice. This resolves a bug I've noticed with our new `LumiComponent` `Button` -- using `R.text` with the `line-height: 20px` was pushing it out of the button (so we were forced to input `T.body_` with an added explicit `line-height`